appveyor-ubuntu.yml

```diff
@@ -23,6 +23,7 @@ environment:
   NODE_VERSION: "14.17.6"
   AWS_S3: 'AWS_S3_TESTING'
   AWS_ECR: 'AWS_ECR_TESTING'
+  CARGO_LAMBDA_VERSION: "v0.17.1"
```

Comment:
Should we run the cargo related tests in GHA instead?

Reply:
That's a good idea. We might still want to keep cargo tests running in the Canary though (i.e. appveyor-ubuntu.yml and appveyor-windows.yml). But moving those running per PR will prevent each PR from taking longer for the checks.

Reply:
Made a change to use GH Actions to run Rust integration tests but there are some caveats:
- The GH Actions now can only run on `ubuntu-latest` but not Windows. Reason is I couldn't find a way to switch Windows Docker to use Linux Containers (unlike AppVeyor).
- Therefore, we still need AppVeyor to run these tests in Windows, which is defined in `appvyeor-windows-build-rust.yml`
- We still run the Rust integration tests in Canary (defined in appveyor-ubuntu.yml and appveyor-windows.yml) to keep the completeness of tests in Canary
- The result is - we only remove running Rust tests in the Appveyor make pr check. But the benefit is significant - we avoid adding an extra 22minutes of time to the Appveyor make pr check.
- Downside - it is a bit confusing as tests are spread across in different CI scripts. I'm a bit worried about the possibility of us missing some tests by having none of the CI scripts to pick up these tests.

Tagging @moelasmar as well as he is working on SAM CLI release improvement. For this PR, I'm in favor of running the Rust integration tests in GH Actions to not add extra time in AppVeyor make pr check. But let's keep the downside in mind and be sure to address it when we revisit this topic in SAM CLI release improvement

Reply:
> it is a bit confusing as tests are spread across in different CI scripts. I'm a bit worried about the possibility of us missing some tests by having none of the CI scripts to pick up these tests

Could we make it a single command (within `Makefile`) to make this easier?